src/App.tsx

```diff
@@ -49,12 +126,19 @@ export const App: React.FC = () => {
               'tile',
               'is-parent',
               'is-8-desktop',
-              'Sidebar',
-              'Sidebar--open',
+              'Sidebar', {
+                'Sidebar--open': selectedPost,
+              },
             )}
           >
             <div className="tile is-child box is-success ">
-              <PostDetails />
+              {selectedPost && (
+                <PostDetails
+                  title={selectedPost.title}
+                  body={selectedPost.body}
+                  id={selectedPost.id}
+                />
```

Comment:
```suggestion
                <PostDetails post={selectedPost} />
```
just pass the entire post object